pkg/services/sqlstore/user.go

```diff
@@ -620,6 +622,21 @@ func (ss *SQLStore) SearchUsers(ctx context.Context, query *models.SearchUsersQu
 			whereParams = append(whereParams, query.OrgId)
 		}
 
+		// user only sees the users in which it has user read access too
+		if ss.Cfg.IsFeatureToggleEnabled(featuremgmt.FlagAccesscontrol) && query.SignedInUser != nil {
+			acFilter, err := accesscontrol.Filter(
+				query.SignedInUser,
+				"u.id",
+				"users:id:",
+				accesscontrol.ActionOrgUsersRead,
```

Comment:
I wanted to use `accescontrol.ActionUsersRead` however when using that permission, no users are shown.